gradle.properties

```diff
@@ -1,9 +1,10 @@
-group=com.marklogic
-version=1.2.2
-
-# For the Confluent Connector Archive
-componentOwner=marklogic
-componentName=kafka-marklogic-connector
-
-# Override this in gradle-local.properties
-kafkaHome=
+group=com.marklogic
+
+version=1.2.3
+
+# For the Confluent Connector Archive
+componentOwner=marklogic
+componentName=kafka-marklogic-connector
+
+# Override this in gradle-local.properties
+kafkaHome=

```

Comment:
Please revert the changes to this file. I'll update the version when I build the next version.

Reply:
Can you help on how to revert ?

Reply:
If you haven't setup a remote that points to the marklogic-community version, setup that remote first:
git remote add upstream https://bgeorge111:<YourGithubPassword>@github.com/marklogic-community/kafka-marklogic-connector.git

Now you can pull the version of the file in that repo:
git fetch --all
git checkout upstream/master -- gradle.properties
git add gradle.properties
git commit -m "reverting gradle.properties"
git push

Reply:
I believe I have done it. https://github.com/marklogic-community/kafka-marklogic-connector/pull/36/commits/b97f9d2e8d4d23c9355b1fa646c6d20c59be9b2d